dimod/constrained.py

```diff
@@ -93,13 +93,14 @@ def vartype(self, v: Variable) -> Vartype:
 class ConstrainedQuadraticModel:
     r"""A constrained quadratic model.
 
-    Constrained quadratic models are problems of the form:
+    Constrained quadratic models represent problems of the form:
```

Comment:
I would say ```:class:`ConstrainedQuadraticModel`\s represent```, but `Constrained quadratic models are`

Reply:
I tend to separate the user-centric perspective in which these models represent (model) real-world user problems such as a knapsack problem from the solver-centric perspective in which these mathematical formulae are the problems submitted to a solver.
Not crucial.

Reply:
Sure, but in this case "constrained quadratic model" is the label that we are applying to that mathematical definition. It's not representing it. The class is arguably a representation because it's not an abstract mathematical definition.